template/page-studs/index/with-tw.tsx

```diff
@@ -43,6 +55,20 @@ const Home: NextPage = () => {
   );
 };
 
+const Headline: React.FC<{ color?: ColorToken; children: React.ReactNode }> = ({
+  color = "gray",
+  children,
+}) => (
+  <h1
+    className={[
+      "text-5xl md:text-[5rem] leading-normal font-extrabold",
```

Comment:
Just to make sure, do these operations affect tailwind CSS intelligence, since the class Names aren't direct values of `className`

Reply:
Don't know about others, but TailwindCSS extension for VS Code does manage to give suggestions

<img width="726" alt="Screenshot 2022-08-07 at 5 27 00 PM" src="https://user-images.githubusercontent.com/48997634/183289348-40761daf-394f-4201-a486-af16c4f10ead.png">